site/layouts/shortcodes/include.html

```diff
@@ -1,6 +1,16 @@
-{{ $file := .Get 0 }}
-[{{$file}}]({{printf "/examples/%s" $file}})
-{{ $path := (printf "static/examples/%s" $file)}}
-```{{ .Get 1 }}
+{{ $file := .Get "filename" }}
+{{ $codelang := .Get "codelang"}}
+{{ $path := (printf "static/examples/%s/%s" $codelang $file )}}
+{{$ghlink := printf "/examples/%s/%s" $codelang $file}}
```

Comment:
codelang will not necessarily be part of the filename.

See #1078, for example

Reply:
Thanks! I will change this part. you are right!